pkgs/top-level/all-packages.nix

```diff
@@ -30623,6 +30623,8 @@ in
 
   wacomtablet = libsForQt5.callPackage ../tools/misc/wacomtablet { };
 
+  wallgen = callPackage ../applications/misc/wallgen { };
```

Comment:
```suggestion
  wallgen = pythonPackages.callPackage ../applications/misc/wallgen { };
```

Reply:
I also changed `pythonPackages.callPackage` to `python3Packages.callPackage`